src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -119,9 +145,16 @@ export default function ActionsPanel(): JSX.Element {
             setActionData(actions.results[0].actions);
           }
           setFetchingActionData(false);
+          setInlineErrors((prevInlineErrors) => [null, prevInlineErrors[1]]);
           return;
         })
-        .catch((error) => console.error(Label.GET_ACTIONS_ERROR, error));
+        .catch((error) => {
+          setInlineErrors((prevInlineErrors) => [
```

Comment:
Here also we need to only update a single error slot so we don't accidentally remove errors if the number of errors in the tuple changes.